scripts/zkai_d128_full_block_accumulator_backend_gate.py

```
@@ -0,0 +1,892 @@
+    source = _validate_source_descriptor(payload)
+    artifact = require_object(payload.get("accumulator_artifact"), "accumulator artifact")
+    verify_accumulator_artifact(artifact, source)
+    handle = require_object(payload.get("verifier_handle"), "verifier handle")
+    verify_verifier_handle(handle, artifact)
+    _validate_recursive_status(payload)
+    expect_equal(payload.get("non_claims"), NON_CLAIMS, "non-claims")
+    expect_equal(payload.get("validation_commands"), VALIDATION_COMMANDS, "validation commands")
+    return source, artifact, handle, _expected_summary(source, artifact, handle)
+
+
+def _validate_draft_payload(payload: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `payload`

(ANN401)